_life/money.md

```diff
@@ -42,29 +55,8 @@ for 2025, your annual salary for that year will be (from above) 54,883 + 1% =
 55,432 CHF. If there is then a 2% adjustment for 2026, your annual salary for
 2026 will be (55,432 + 1,000) + 2% = 57,561 CHF.
 
-There is one slightly unusual factor to take into account: the 13th salary.
-The 13th salary system means that you get 1/13th of your annual salary each month,
-with the last 1/13th being given to you partly in November and partly in December.
-The origins of this system are historically not so clear, but nowadays it’s basically a way for you to pay for Christmas presents and such.
 
-Overall, expect to get around 3,200 CHF/month after all taxes starting in the first year, plus the 13th salary.
-Here’s an example breakdown for a first year monthly PhD student salary, for a non-November/December month (see above), **as of January 2023** (the amounts may have slightly changed since then):
-
-| Gross salary             | 4194.65 CHF |
-| First pillar (AVS)       | -222.30 CHF  |
-| Various other deductions | -83.75 CHF   |
-| Second pillar (Publica)  | -152.70 CHF  |
-| Income tax               | -251.25 CHF  |
-| **Net income**           | **3484.65 CHF** |
-
-
-## Banks
-
-Go with whichever Swiss bank you want, they're all more or less equal, and will have free accounts for students. The main question is whether you want a more traditional account or an app-based one like CSX or Yuh.
```

Comment:
Do you think it's worth keeping a sentence (either here or in the summary at the top) about saving a few francs with an app-only bank?

Reply:
done